Cannot find module 'graphql-query-complexity'

Hello,

With the latest `0.12.0-rc.10`, we are getting the following error during our TypeScript check even though we are not using this plugin.

`node_modules/nexus/dist/plugins/queryComplexityPlugin.d.ts(2,41): error TS2307: Cannot find module 'graphql-query-complexity'.`

Thank you for all the work!

--- a/package.json
+++ b/package.json
@@ -78,7 +78,6 @@
     "eslint": "^6.6.0",
     "graphql": "^14.5.8",
     "graphql-iso-date": "^3.6.1",
-    "graphql-query-complexity": "^0.4.1",
     "graphql-relay": "^0.6.0",
     "husky": "^1.1.2",
     "jest": "^24.9.0",

--- a/src/plugins/queryComplexityPlugin.ts
+++ b/src/plugins/queryComplexityPlugin.ts
@@ -1,7 +1,6 @@
 import { plugin } from "../plugin";
 import { printedGenTyping, printedGenTypingImport } from "../utils";
 import { RootValue, ArgsValue, GetGen } from "../core";
-import { ComplexityEstimatorArgs } from "graphql-query-complexity";
 import { GraphQLField } from "graphql";
 
 const QueryComplexityImport = printedGenTypingImport({
@@ -24,7 +23,7 @@ const fieldDefTypes = printedGenTyping({
 export type QueryComplexityEstimatorArgs<
   TypeName extends string,
   FieldName extends string
-> = ComplexityEstimatorArgs & {
+> = {
   type: RootValue<TypeName>;
   field: GraphQLField<
     RootValue<TypeName>,
@@ -32,6 +31,7 @@ export type QueryComplexityEstimatorArgs<
     ArgsValue<TypeName, FieldName>
   >;
   args: ArgsValue<TypeName, FieldName>;
+  childComplexity: number;
 };
 
 export type QueryComplexityEstimator<

